Makefile

```diff
@@ -0,0 +1,22 @@
+TOOLS = docker-compose run --rm tools
+CONSOLE = ./bin/console
```

Comment:
Yes, later, this was a quick enhancement, not a Docker integration which is out of scope.